docs/content/topics/scripting/guide.md

```diff
@@ -9,1401 +9,867 @@ aliases:
-
-
-  var first = new Group()
-  first.add(hello)
-  first.add(goodbye)
-
-  var second = new Group()
-  second.add(helloAgain)
-  second.add(goodbyeAgain)
-
-  first.runAll().then( () => second.runAll() )
-})
+events.process();
 ```
-[brigade-12.js](../../examples/brigade-12.js)
-
-The above creates two groups, and then later executes them. The order of execution would
-be:
 
-- Run both of the jobs in the `first` group.
-- Once those two jobs have both completed, run both of the jobs in the `second` group.
-
-This is the way groups can be used to control the order in which groups of jobs are run.
-
-So far we have looked at the brigade.js file, the event registry, and jobs and groups. As we
-advance our way through Brigade, we will next take a look at how `brigade.js` scripts can work
-with the BrigadeEvent and Project objects that are sent to every event handler.
+Being able to associated a Git repository to a project is a convenient way to
+provide version-controlled data to our Brigade scripts. And it makes Brigade a
+great tool for executing CI pipelines, deployments, packaging tasks, end-to-end
+tests, and other DevOps tasks.
 
 ## Working with Event and Project Data
 
-Two pieces of information are passed into every event handler: The event that occurred,
-and the project.
+As we've seen, the event object is always passed into an event handler. This
+object also includes project data. Let's look at the data we have access to.
 
 ### The Brigade Event
 
-From the event, we can find out what triggered the event, what data was sent with the
-event, and (if there is a repository) what repository commit we should be using.
-
-An event looks like this:
-
-```javascriot
-var e = {
-  buildID: "brigade-worker-01brwzs64rve2jvky87hxy1wsp-master",
-  type: "brig",
-  provider: "brig",
-  commit: "master",
-  payload: ""
-}
-```
-
-- `buildID` is a unique per-build ID. Every time a new build is triggered, a new ID will
-  be generated.
-- `type` is the event type. A GitHub Pull Request, for example, would set type to
-  `pull_request`.
-- `provider` tells what service triggered this event. A GitHub request will set this to
-  `github`.
-- `commit` is the commit (revision ID) for the VCS. For Git, this can be a SHA, a branch
-  name, or a tag.
-- `payload` contains any information that the hook sent when triggering the event. For
-  example, a GitHub push request generates [a rather large payload](https://developer.github.com/v3/activity/events/types/#pushevent). Payload is an unparsed string.
-
-When one event triggers another event, the triggered event also gets information about the
-thing that caused it. This is available on the `event.cause` field. The `after` and
-`error` events, for example, will be able to access the `cause` field and see the event
-that triggered them.
+From the event, we can find out what triggered the event, what data was sent
+with the event, and (if there is a repository) what repository commit we should
+be using.
```

Comment:
Is the question of which commit relevant to someone writing a script?

By the time the script executes, the git initializer for the worker has already worked out what commit to use. Each job's git initializer (if applicable) will work that out as well-- all without the script author needing to be concerned with this.